webofneeds/won-cryptography/src/main/java/won/cryptography/rdfsign/WonVerifier.java

```diff
@@ -15,6 +17,7 @@
 import won.protocol.util.WonRdfUtils;
 import won.protocol.vocabulary.WONMSG;
 
+import java.io.StringWriter;
```

Comment:
this message seems to print a warning now in the owner logs:

9.8.16 12:57:59.350 [Camel (wonOwnerCamel) thread #18 - JmsConsumer[OwnerProtocol.Out.257ec8ad1128a9f6d4f4a5ca77356fffa04d457b180f2762a7869d8f]] WARN  won.cryptography.rdfsign.WonVerifier - cannot verify signature https://satsrv06.researchstudio.at/won/resource/event/6631698687093502000#envelope-l6rb-sig as it is not part of this message
9.8.16 12:58:00.041 [Camel (wonOwnerCamel) thread #14 - JmsConsumer[OwnerProtocol.Out.257ec8ad1128a9f6d4f4a5ca77356fffa04d457b180f2762a7869d8f]] WARN  won.cryptography.rdfsign.WonVerifier - cannot verify signature https://satsrv06.researchstudio.at/won/resource/event/rr8eomkdztufirwvuhqm#envelope-xz3q-sig as it is not part of this message
...

i was creating two needs, got some matches and created a connection and exchaned a few messages. => Fix this warning messages